assets/javascripts/lib/discourse-markdown/discourse-calendar.js.es6

```diff
@@ -1,30 +1,81 @@
 const calendarRule = {
   tag: "calendar",
 
-  wrap(token, info) {
-    token.attrs = [
+  before: function(state, info) {
+    let wrapperDivToken = state.push("div_open", "div", 1);
+    wrapperDivToken.attrs = [["class", "discourse-calendar-wrap"]];
+
+    // div.discourse-calendar-header
+    let headerDivToken = state.push("div_open", "div", 1);
+    headerDivToken.attrs = [["class", "discourse-calendar-header"]];
+
+    let titleH2Token = state.push("h2_open", "h2", 1);
+    titleH2Token.attrs = [["class", "discourse-calendar-title"]];
+    state.push("h2_close", "h2", -1);
+
+    let timezoneWrapToken = state.push("span_open", "span", 1);
+    timezoneWrapToken.attrs = [
+      ["class", "discourse-calendar-timezone-wrap"]
+    ]
+    if (info.attrs.tzpicker === "true") {
+      _renderTimezonePicker(state, info);
+    }
+    state.push("span_close", "span", -1);
+
+    state.push("div_close", "div", -1);
+    // end div.discourse-calendar-header
+
+    // div.calendar
+    let mainCalendarDivToken = state.push("div_open", "div", 1);
+    mainCalendarDivToken.attrs = [
       ["class", "calendar"],
       ["data-calendar-type", info.attrs.type || "dynamic"],
-      ["data-calendar-default-view", info.attrs.defaultView || "month"]
+      ["data-calendar-default-view", info.attrs.defaultView || "month"],
+      ["data-calendar-default-timezone", info.attrs.tzdefault]
```

Comment:
maybe `info.attrs.defaultTimezone` would be more consistent with the data attribute name and what we do for `defautView`